tests/test_catalyst.py

```diff
@@ -0,0 +1,93 @@
+import unittest
+import collections
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+import torchvision
+import torchvision.transforms as transforms
+
+import catalyst
+from catalyst.dl import SupervisedRunner, CheckpointCallback
+from safitty import Safict
+
+
+class TestCatalyst(unittest.TestCase):
+
+    def test_version(self):
+        self.assertIsNotNone(catalyst.__version__)
+    
+    def test_mnist(self):
+        bs = 32
+        num_workers = 4
+
+        data_transform = transforms.Compose([
+            transforms.ToTensor(),
+            transforms.Normalize((0.1307, ), (0.3081, ))
+        ])
+
+        loaders = collections.OrderedDict()
+
+        trainset = torchvision.datasets.MNIST(
+            root='./data', train=False,
+            download=True, transform=data_transform)
```

Comment:
The tests don't have internet access (they are hermetic), but you can use `np.random` to generate a train set:
https://github.com/Kaggle/docker-python/blob/master/tests/test_keras.py#L34

Reply:
@rosbo have we resolve the conversation?